client/components/action-card/style.scss

```diff
@@ -30,7 +30,7 @@
 }
 
 .action-card__heading {
-	font-size: 21px;
+	font-size: 20px;
```

Comment:
I think we should stick with 21, 20 isn't one of the sizes we use, according to the handbook: https://wordpress.com/design-handbook/typography/

Reply:
My goal was to give a different size to headings on the Select Business Type page, as otherwise we're losing a sense of hierarchy:

<img width="470" alt="screenshot" src="https://user-images.githubusercontent.com/594356/38138653-41d6b344-342c-11e8-86d4-9d4c04aedcb6.png">

I'll use 24px for the main heading, and 21px for the others then.